src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -548,7 +548,18 @@ public function process_payment( $order_id ) {
 					Mollie_WC_Plugin::debug( 'Creating payment object: type Order, first try creating a Mollie Order.' );
 
 					// Only enable this for hardcore debugging!
-					// Mollie_WC_Plugin::debug( $data );
+                    $apiCallLog = [
+                            'amount'=>$data['amount'],
```

Comment:
To avoid undefined index notice, check if the key exist:
```suggestion
                            'amount' => isset($data['amount']) ? $data['amount'] : '',
```